src/server/main_service.cc

```diff
@@ -2434,7 +2434,7 @@ void Service::RegisterCommands() {
   registry_.Init(pp_.size());
 
   using CI = CommandId;
-  if (VLOG_IS_ON(1)) {
+  if (VLOG_IS_ON(2)) {
```

Comment:
it outputs 100 lines of what commands exist in the CI, I don't think that's really useful